apps/rush-lib/src/cli/actions/ProjectsAction.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseRushAction } from './BaseRushAction';
+import { RushCommandLineParser } from '../RushCommandLineParser';
+
+export class ProjectsAction extends BaseRushAction {
+  constructor(parser: RushCommandLineParser) {
+    super({
+      actionName: 'projects',
```

Comment:
It might be more natural to have the action verb be `list` and eventually have several different categories that can be listed.

Like `rush list projects`, `rush list version-policies`, etc.

Reply:
@iclanton FYI I added some other design thoughts to issue https://github.com/Microsoft/web-build-tools/issues/1242

If we make the command `list`, then **ts-command-line** requires the kind of thing to be specified as a command-line parameter (e.g. `rush list --items projects --field version`).  This is okay... but we should be careful not to overengineer it too much.

Keep in mind that there is an extremely flexible alternative that requires relatively little effort from the user:
```shell
C:\MyRepo>node
> let rushLib = require("@microsoft/rush-lib")
> let config = rushLib.RushConfiguration.loadFromDefaultLocation()
> config.projects.map(x => x.packageName)
```
So we really want `rush list` to be as simple and intuitive as possible.

Reply:
i already added --field feature, would add --items too.
will update the code today.
Should i verify if --field value is valid or not based on --item?

Reply:
Yes. I would also suggest that `--items` should default to projects if it is omitted.

Reply:
So. I took ideas from @octogonz and @ianc and came up with the following API:
`rush list` (lists all projects)
`rush list --version` (lists projects and version)
`rush list --path` (lists projects and path)
`rush list --full-path` (lists projects and full path)
And it lets you do any combination, along with short form, so e.g.,
`rush list -vp` produces:
<img width="1072" alt="Screen Shot 2019-06-20 at 9 51 05 PM" src="https://user-images.githubusercontent.com/5139752/59898438-9d552280-93a5-11e9-9d53-b0908e10917a.png">